test/test_matmul_cuda.py

```diff
@@ -150,7 +149,8 @@ def test_cublas_addmm_alignment(self, dtype):
 
     @onlyCUDA
     @unittest.skipIf(IS_JETSON, "Too large for Jetson")
-    @toleranceOverride({torch.float32: xtol(atol=1e-5, rtol=1.1e-5)})
+    @toleranceOverride({torch.float32: xtol(atol=1e-5, rtol=1.1e-5),
```

Comment:
Do we know why we need to increase these tolerances for ROCM?

Reply:
We need rocblas provides the capability to disallow low precision kernels. So, we could possibly share the same tolerances as the cuda backend for the test case.

A good suggestion but it probably outside the scope of this PR.

Reply:
The expectation in terms of precision in PyTorch for the default is that we never use low precision kernels by default.
If rocblas doesn't have such setting, my expectation is that we wouldn't use rocblas for now to have the expected behavior

Reply:
Thats very clear, we will revisit this PR when the backend selector is ready.